pod/perluniintro.pod

```diff
@@ -398,10 +398,14 @@ and on already open streams, use C<binmode()>:
 The matching of encoding names is loose: case does not matter, and
 many encodings have several aliases.  Note that the C<:utf8> layer
 must always be specified exactly like that; it is I<not> subject to
+<<<<<<< HEAD
 the loose matching of encoding names. Also note that currently C<:utf8> is unsafe for
 input, because it accepts the data without validating that it is indeed valid
 UTF-8; you should instead use C<:encoding(UTF-8)> (with or without a
 hyphen).
+=======
+the loose matching of encoding names.
+>>>>>>> f7eed95508 (Made :utf8 an actual layer)
```

Comment:
some merge conflict detritus

Reply:
Yeah rebasing a 12 years old branch was a hell, I thought I had fixed everything but apparently not.